tasks/plugins.mjs

```diff
@@ -19,17 +19,6 @@ const iconsPlugins = [
       attrs: "(id|class|data-name|fill|width|height|viewBox|fill-rule)"
     }
   },
-  {
-    name: 'addAttributesToSVGElement',
-    params: {
-      attributes: [
-        `viewBox="0 0 24 24"`,
-        {
-          'aria-hidden': true,
-        }
-      ],
-    }
-  }
```

Comment:
This has changed to be local to the `icons react` function so that we can still share the rest of the plugins between the optimise and the react generator code.

As we don't want to add aria-hidden to the raw svgs as this is not supported functionality for svgs